app/components/SubscriptionScheduleClass.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import {connect} from 'react-redux';
+
+
+export const SubscriptionSchedule = () => (
+    <div>
```

Comment:
We can later optimize this by creating a checkbox component and pass down days as props, so we don't repeat ourselves. We can add this as an issue for later.